CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.24.20] - 2022-02-17
+
+### Added
+
+- Added how to create custom executors
+
+### Changed
+
+- Changed the description of the hyperlink for choosing executors
+- Fixed typos in doc.source.api.getting_started.how_to.execution.creating_custom_executors.ipynb
```

Comment:
Can you change it to this instead: `doc/source/api/getting_started/how_to/execution/creating_custom_executors.ipynb`?